### ISSUE TITLE
Indication that something is not selectible.

### ISSUE THREAD

**Issue by jaweber1** [FrSky-OS/discussion#61]
I think that I've figured out that if something is light grey, it is not editable.  This was not totally obvious. It would be better if it was just gone if it was blank and not editable. Or if non-blank, then it should look different than the editable field and just show the value.

**Comment by yellowbird911** [comment]
May be a minor issue to most but almost 10% of the male population has some form of color blindness of some form.  When radio manufactures create menus that use color queues instead of a more visible queues it makes for a poor experience.  The Walkera Devo12s did this as was very hard for me to use the GUI.

**Comment by AloftHobbies** [comment]
Yes, I would like this to be a higher contrast to make it clear when something is disabled.
